docs/kqueen.cli_gcloud_examples.rst

```diff
@@ -0,0 +1,150 @@
+
+   $ git clone https://github.com/Mirantis/kqueen.git
+   $ cd kqueen
+   $ docker-compose -f docker-compose.yml -f docker-compose.demo.yml rm -f # Make sure you are starting from scratch
+   $ docker-compose -f docker-compose.yml -f docker-compose.demo.yml up
+   $ TOKEN=$(curl -s -H "Content-Type: application/json" --data '{ "username": "admin", "password": "default" }' -X POST 127.0.0.1:5000/api/v1/auth | jq -r '.access_token')
+   $ echo $TOKEN
+
+#. Create new organization "testorganization" with new user/password "testusername / testpassword":
+
+.. code-block:: bash
+
+   $ curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" --data '{ "name": "testorganization", "namespace": "testorganization" }' -X POST 127.0.0.1:5000/api/v1/organizations | jq
+   $ ORG_ID=$(curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json"  127.0.0.1:5000/api/v1/organizations | jq -r '.[] | select (.name == "testorganization").id')
+   $ echo $ORG_ID
+   $ curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" --data "{ \"username\": \"testusername\", \"organization\": \"Organization:$ORG_ID\", \"role\": \"superadmin\", \"active\": true, \"password\": \"testpassword\" }" -X POST 127.0.0.1:5000/api/v1/users | jq
+
+#. Switch to new user "testusername" and add new Google Cloud Kubernetes Service provisioner:
+
+.. code-block:: bash
+
+   $ TOKEN=$(curl -s -H "Content-Type: application/json" --data '{ "username": "testusername", "password": "testpassword" }' -X POST 127.0.0.1:5000/api/v1/auth | jq -r '.access_token')
+   $ echo $TOKEN
+   $ USER_ID=$(curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" 127.0.0.1:5000/api/v1/users | jq -r '.[] | select (.username == "testusername").id')
+   $ echo $USER_ID
+   $ SERVICE_ACCOUNT_INFO=$(cat ../key.json)
+   $ echo $SERVICE_ACCOUNT_INFO
+   $ curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" --data "{ \"name\": \"testprovisioner\", \"engine\": \"kqueen.engines.GceEngine\", \"owner\": \"User:$USER_ID\", \"parameters\": { \"project\": \"$GCE_PROJECT_ID\", \"service_account_info\": $SERVICE_ACCOUNT_INFO } }" -X POST 127.0.0.1:5000/api/v1/provisioners | jq
+
+#. Deploy Kubernetes cluster using the GKE provisioner:
+
+.. code-block:: bash
+
+   $ PROVISIONER_ID=$(curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" 127.0.0.1:5000/api/v1/provisioners | jq -r '.[] | select (.name == "testprovisioner").id')
+   $ echo $PROVISIONER_ID
+   $ curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" --data "{ \"name\": \"testcluster\", \"owner\": \"User:$USER_ID\", \"provisioner\": \"Provisioner:$PROVISIONER_ID\", \"metadata\": { \"machine_type\": \"n1-standard-1\", \"node_count\": 1, \"zone\": \"us-central1-a\" } }" -X POST 127.0.0.1:5000/api/v1/clusters | jq
+
+#. Check the status of the cluster by query KQueen API (run this command multiple times):
```

Comment:
Maybe it's better to specify the expected output which signifies that status is ok?

Reply:
I will add the proper output later once I'll do more testing...